tests/integration_tests/performance/test_huge_pages.py

```diff
@@ -246,8 +246,16 @@ def test_ept_violation_count(
         # Give the helper time to touch all its pages
         time.sleep(5)
 
+        if global_props.cpu_architecture == "x86_64":
+            trace_entry = "reason EPT_VIOLATION"
+        else:
+            # On ARM, KVM does not differentiate why it got a guest page fault.
+            # However, even in this slightly more general metric, we see a significant
+            # difference between 4K and 2M pages.
+            trace_entry = "guest_page_fault"
+
         _, ept_violations, _ = utils.run_cmd(
-            "cat /sys/kernel/tracing/trace | grep 'reason EPT_VIOLATION' | wc -l"
+            f"cat /sys/kernel/tracing/trace | grep '{trace_entry}' | wc -l"
         )
 
     metrics.put_metric("ept_violations", int(ept_violations), "Count")
```

Comment:
I think in this case it would be nice to have metrics with different names. Otherwise it would still be "ept_violations" on ARM.

Reply:
mh, true

Reply:
changed